tests/0097-ssl_verify.cpp

```diff
@@ -34,6 +34,43 @@
 #include "testcpp.h"
 #include "tinycthread.h"
 
+static const std::string envname[RdKafka::CERT__CNT][RdKafka::CERT_ENC__CNT] =
+  {
+   /* [RdKafka::CERT_PUBLIC_KEY] = */
+   {
+    "RDK_SSL_pkcs",
+    "RDK_SSL_pub_der",
+    "RDK_SSL_pub_pem",
+   },
+   /* [RdKafka::CERT_PRIVATE_KEY] = */
+   {
+    "RDK_SSL_pkcs",
+    "RDK_SSL_priv_der",
+    "RDK_SSL_priv_pem",
+   },
+   /* [RdKafka::CERT_CA] = */
+   {
+    "RDK_SSL_pkcs",
+    "RDK_SSL_ca_der",
+    "RDK_SSL_ca_pem",
+   }
+  };
+
+
+static std::vector<char> read_file (const std::string path) {
```

Comment:
is this generally useful enough to put somewhere common?

Reply:
rule of nineteen applies here.